src/figuretype/docker.c

```diff
@@ -85,13 +86,13 @@ static int try_export_resource(int building_id, int resource, int city_id)
     return 0;
 }
 
-static int get_closest_warehouse_for_import(int x, int y, int city_id, int distance_from_entry, int road_network_id,
+static int get_closest_warehouse_for_import(int x, int y, int city_id, building *dock,
```

Comment:
Nitpick: `const building *dock` (and do the same to `get_closest_warehouse_for_export`

Reply:
the problem now is all functions that accepts `building *` complain when I pass `const *building` @crudelios what do you think?

Reply:
I may have missed that. If functions inside this one require `building *` instead of `const building *`, then this one should be kept as `building *` as well.